commonknowledge/join
# Add support for Stripe, as yet unimplemented

This adds a new block called Minimal Join Form, which represents the shorter version of the Join Form that is intended for the Pelican House website.

We generally want to support Stripe and Pelican House in particular wants to use Stripe, so we add Stripe support to only this Minimal Join Form block for now. So far the support for this means passing variables through from WordPress to React, and nothing further.

On the React side, nothing happens yet. Adding the Minimal Join Form to any WordPress post or page won't do anything.

Contributes to CK-3804.

## Motivation and Context




Advances the Stripe feature but in a way that doesn't change anything. As we don't get have continuous deployment for this project, this is a safe change to make.

## How Can It Be Tested?




1. Go to a page.
2. Add the Minimal Join Form to a page.
3. Observe that when saved on the page you see in the HTML:
```html
<div class="ck-minimalist-join-flow ck-join-form">
  <div class="ck-minimalist-join-form"></div>
 </div>
```
This is where the React application will attach.

## How Will This Be Deployed?



Standard process.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:


- My change requires a change to the documentation.
- I've updated the documentation accordingly.
